cppcache/integration-test/ThinClientTransactions.hpp

```diff
@@ -632,7 +632,7 @@ DUNIT_TASK_DEFINITION(CLIENT1, SuspendResumeCommit)
     bool resumeExc = false;
     try {
       txManager->resume(m_suspendedTransaction);
-    } catch (const IllegalStateException) {
+    } catch (const IllegalStateException &e) {
```

Comment:
Haven't verified yet, but I'm fairly certain naming the variable here will generate a new warning on Clang.  I believe adding the '&' but leaving it anonymous works.

Reply:
Can confirm, this breaks our clang build with an error "unused exception parameter 'e'".  Please leave these anonymous wherever they're not used.